cranelift/filetests/filetests/isa/x64/struct-arg.clif

```diff
@@ -130,14 +130,13 @@ block0(v0: i64, v1: i64, v2: i64):
 ;   virtual_sp_offset_adjust 192
 ;   lea     0(%rsp), %rdi
 ;   movl    $128, %edx
-;   load_ext_name %Memcpy+0, %rcx
-;   call    *%rcx
+;   load_ext_name %Memcpy+0, %rax
+;   call    *%rax
 ;   lea     128(%rsp), %rdi
-;   movq    %r14, %rsi
```

Comment:
This is one of the moves removed by this PR, and is redundant with the move on line 139.